Classes/CocoaLumberjack.swift

```diff
@@ -107,6 +106,9 @@ public func DDLogError(_ message: @autoclosure () -> String, level: DDLogLevel =
     _DDLogMessage(message, level: level, flag: .error, context: context, file: file, function: function, line: line, tag: tag, asynchronous: async, ddlog: ddlog)
 }
 
+// swiftlint:disable identifier_name
+// swiftling doesn't like func names that begin with a capital letter
```

Comment:
I think we should still rename the function and just introduce a deprecated version with the old naming (similar to what I did when renaming `defaultDebugLevel` to `dynamicLogLevel`). SwiftLint isn't wrong that identifiers should be lowerCamelCase. But if we just change it, there's not smooth upgrade path. or would you rather do that in a subsequent PR?

Also, there's a small typo in this line "swiftling" 😉

Reply:
Ok, will do.